tests/test_ocs_basic_install.py

```diff
@@ -34,7 +35,8 @@ def run(**kwargs):
     log.info("Generating install-config")
     # TODO: determine better place to create cluster directories - (log dir?)
     cluster_dir_parent = "/tmp"
-    base_name = test_data.get('cluster-name', 'ocs-ci-cluster')
+    default_cluster_name = f"ocs-ci-cluster-{getuser()}"
```

Comment:
I think we can probably remove 'cluster' from the default string at this point. `ocs-ci-<username>` seems sufficient to me, and the name does get a bit unwieldy once we add our unique number as well as the unique string the installer adds.